google_deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
run this script to deploy your current workspace to google app engine.

this is not a permanent solution.